libethereum/State.cpp

```diff
@@ -546,18 +546,17 @@ void State::rollback(size_t _savepoint)
 
 std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _envInfo, SealEngineFace const& _sealEngine, Transaction const& _t, Permanence _p, OnOpFunc const& _onOp)
 {
-    auto onOp = _onOp;
-#if ETH_VMTRACE
-    if (isChannelVisible<VMTraceChannel>())
```

Comment:
This check is not needed anymore, because in the logging statement like
```
        BOOST_LOG(traceLogger) << dumpStackMemoryStorage(ext, vm);
```
It doesn't call `dumpStackMemoryStorage(ext, vm)` at all when current log filters block all `traceLogger` messages.